src/scss/components/optional/_order-card.scss

```diff
@@ -112,6 +112,10 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
 
     .c-orderCard-title {
         line-height: line-height(16px, 24px);
+        font-family: $font-family-headings;
+        font-weight: $font-weight-base;
+        font-size: 22px;
```

Comment:
Should use the `@include font-size()` mixin so that this gets converted to `rem`s